lib/chewy.rb

```diff
@@ -14,19 +15,19 @@
 require 'active_support/core_ext/object/inclusion'
 require 'active_support/core_ext/string/inflections'
 
-require 'i18n/core_ext/hash'
-require 'chewy/backports/deep_dup' unless Object.respond_to?(:deep_dup)
-require 'singleton'
-require 'base64'
-
-require 'elasticsearch'
-
 def try_require(path)
   require path
 rescue LoadError
   nil
 end
 
+try_require('i18n/core_ext/hash')
+require 'chewy/backports/deep_dup' unless Object.respond_to?(:deep_dup)
```

Comment:
This looks like another backport from rails 4 for rails 3.2 support.  Can it be removed?